### PR TITLE
[ATX-2024] Kickoff

### PR DESCRIPTION
**Pull request by nimbinatus**
Adding starter for Austin ~(draft until organizers settled)~. Ready for review.

### CODE REVIEW

File: static/events/2024-austin/organizers/chris-b.jpg
Comment (jyee) [inline static/events/2024-austin/organizers/chris-b.jpg:1]: @nimbinatus Can you size these down? 1.81MB is rather sizeable. Also worth noting that netlify doesnt create optimized images, so folks loading the page are having to download a lot just for these to get rendered as small thumbnails.

Per the [utilities docs on organizers](https://github.com/devopsdays/devopsdays-web/tree/1d3a7fdbc38d1bde477c4636dea4d61682ee9ddf/utilities#local-organizers), images should be 300px to 600px square.

File: static/events/2024-austin/organizers/karl.jpg
Comment (jyee) [inline static/events/2024-austin/organizers/karl.jpg:1]: Another large image to shrink

File: static/events/2024-austin/organizers/tim.jpeg
Comment (jyee) [inline static/events/2024-austin/organizers/tim.jpeg:1]: scale down

**Review by jyee — CHANGES_REQUESTED** [review]
A few images need to be scaled down as they're quite large. Other than that, the PR is good to merge. I can squash merge when they've been updated, so you don't need to use git replace or other tricks to remove the large images from the repo.

**Comment by FloorD** [comment]
@nimbinatus did you manage to resize the images, or would you like help?

**Comment by nimbinatus** [comment]
> @nimbinatus did you manage to resize the images, or would you like help?

Sorry, just been buried with KubeCon NA things. I'll hopefully get to this today or tomorrow. But thank you for the offer and the reminder!

File: static/events/2024-austin/organizers/tim.jpeg
Comment (nimbinatus) [reply static/events/2024-austin/organizers/tim.jpeg:1]: fixed

File: static/events/2024-austin/organizers/karl.jpg
Comment (nimbinatus) [reply static/events/2024-austin/organizers/karl.jpg:1]: fixed

**Pull request merged**